app/javascript/hooks/queries/admins/useActiveUsers.jsx

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from 'react-query';
+import axios from 'axios';
```

Comment:
The use of the custom `axiosInstance` will be necessary in the upcoming sprints.